app/Http/Controllers/DemandController.php

```diff
@@ -107,22 +108,30 @@ public function viewScript(Request $request)
         $jsPath = public_path('-/view.js');
 
         $response = new StreamedResponse();
-        $response->setCallback(function () use ($jsPath, $request, $params) {
-            echo str_replace([
-                "'{{ ORIGIN }}'",
-            ], $params, file_get_contents($jsPath));
-        });
+        $response->setCallback(
+            function () use ($jsPath, $request, $params) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$request" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1481)